src/Akeneo/Platform/Job/front/process-tracker/src/feature/models/JobExecutionTable.ts

```diff
@@ -21,10 +21,18 @@ type JobExecutionTable = {
   matches_count: number;
 };
 
-const stoppableStatus = ['STARTING', 'STARTED'];
-const jobCanBeStopped = (jobExecutionRow: JobExecutionRow): boolean => {
-  return jobExecutionRow.is_stoppable && stoppableStatus.includes(jobExecutionRow.status);
+const STOPPABLE_STATUS = ['STARTING', 'STARTED'];
+const jobCanBeStopped = (jobExecutionRow: JobExecutionRow): boolean =>
+  jobExecutionRow.is_stoppable && STOPPABLE_STATUS.includes(jobExecutionRow.status);
+
+const RESTRICTED_JOB_TYPES = ['import', 'export'];
+const canShowJobExecutionDetail = (isGranted: (acl: string) => boolean, jobExecutionRow: JobExecutionRow) => {
+  if (RESTRICTED_JOB_TYPES.includes(jobExecutionRow.type)) {
+    return isGranted(`pim_importexport_${jobExecutionRow.type}_execution_show`);
+  }
```

Comment:
It could be more cleaner to do this :
```suggestion

const JOB_TYPES_WITH_ACL = {'import': 'pim_importexport_import_execution_show', 'export': 'pim_importexport_import_execution_show'};
const canShowJobExecutionDetail = (isGranted: (acl: string) => boolean, jobExecutionRow: JobExecutionRow) => {
  if (jobExecutionRow.type in JOB_TYPES_WITH_ACL) {
    return isGranted(JOB_TYPES_WITH_ACL[jobExecutionRow.type]);
  }
```